engines/ultima/ultima8/kernel/object_manager.h

```diff
@@ -66,13 +66,14 @@ class ObjectManager {
 	bool load(Common::ReadStream *rs, uint32 version);
 
 	Object *loadObject(Common::ReadStream *rs, uint32 version);
-	Object *loadObject(Common::ReadStream *rs, Std::string classname, uint32 version);
 
 	Std::vector<Object *> _objects;
 	idMan *_objIDs;
 	idMan *_actorIDs;
 
 private:
+	Object *loadObject(Common::ReadStream *rs, Std::string classname, uint32 version);
```

Comment:
just noticed this can probably be a const reference, but that can be a separate cleanup.